modules/ROOT/pages/alerts-config.adoc

```diff
@@ -5,17 +5,61 @@ endif::[]
+* Advanced alerts for graphs in custom dashboards in Anypoint Monitoring
 +
-Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the General tab for the graph. If the value is above the threshold, the alert is triggered. The alert is triggered only if the alert state changes (from OK state to Alerting state or vice versa).
+Limit per organization: Up to 20 advanced alerts.
++
+Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the *General* tab for the graph. If the value passes the threshold, the advanced alert is triggered. The advanced alert is triggered only if the alert state changes (from `OK` state to `Alerting` state or vice versa).
+* Operational alerts for APIs through API Manager
+* Operational alerts for the runtime through Runtime Manager
+
+[[basic-alerts]]
+== Set up Basic Alerts for Servers and Mule Apps
+
+You can set up basic alerts to trigger email notifications when a metric you are measuring passes a specified threshold.
+
+You can create basic alerts for the following metrics for servers or Mule apps:
+
+* For on-premises servers and CloudHub apps:
+** CPU utilization
+** Memory utilization
+** Thread count
+* For on-premises, Runtime Fabric, and CloudHub apps:
+** Message count
+** Message error count
+** Message response time
+
+=== Configure a Basic Alert
+
+When you create a basic alert, you can select a source for the basic alert, configure the condition under which the basic alert is triggered, and specify which email addresses receive notifications for basic alert triggers.
+
+To create a basic alert:
+
+. Navigate to Anypoint Monitoring.
+. Click the *Alerts* tab.
+. Click the *+ New Alerts* button.
+. Click *Basic alert*.
+. In the *Add basic alert* window, enter the information required to create your basic alert. Note that requirements change depending on the source type you select.
+. Click *Save*.
+
+image::basic-alerts-config.png[Configure a Basic Alert]
 
-== Setting Up API and Runtime Alerts
+[[advanced-alerts]]
+== Set up Advanced Alerts for Graphs in Custom Dashboards
 
-From Alerts in Anypoint Monitoring, you can link out to configuration pages for API and Runtime alerts.
+*https://www.mulesoft.com/anypoint-pricing[Titanium Subscription Required]*
+
+You can set up advanced alerts to trigger email notifications when a data point you are measuring (`JVM - CPU % Utilization`) passes a specified threshold. These advanced alerts are listed in the *Alerts* page in Anypoint Monitoring.
+
+//ANYPOINT MONITORING ALERTS
+include::partial$include-alerts-config.adoc[leveloffset=+1]
+
+== Set Up API and Runtime Alerts
+
+From the Alerts feature in Anypoint Monitoring, you can link out to configuration pages for API and Runtime alerts.
```

Comment:
Is "Alerts feature" capitalized because we have established this as a proper noun, and is it not bold because it does not actually appear in the UI?
Sigh....

Reply:
Also...I would not use "link out," just "link."
And runtime vs. Mule.
```suggestion
From the alerts feature in Anypoint Monitoring, you can link to configuration pages for API and runtime alerts.
```

Reply:
```suggestion
From the *Alerts* page in Anypoint Monitoring, you can link to configuration pages for API Manager and Runtime Manager alerts.
```